### DISCUSSION TITLE
Notifiche circolari/avvisi

### DISCUSSION THREAD

**Discussion by luigirotondi** [iisgiua/giuaschool#700] [Richieste di aiuto]
@trinko Salve, si possono attivare le notifiche delle circolari e avvisi tramite e-mail? I docenti e famiglie hanno attivato dal loro profilo le notifiche ma non arrivano le notifiche al momento della circolare o avviso.
Attendo risposte.

**Comment by trinko** [comment]
Le notifiche tramite circolare normalmente funzionano.
Hai controllato che l'invio delle email viene eseguito in altri casi, ad esempio facendo il test nella pagina della configurazione dell'email?

I possibili problemi possono essere dovuti: 
- indirizzo del mittente: alcuni server di posta bloccano le mail che sono spedite da un sito internet differente dal dominio del mittente; es. mittente qualcosa@gmail.com ma lo spedisci dal sito nome.altervista.org
- se usi la versione free di Altervista è possibile che ci sia un limite al numero di mail inviate: contatta il loro supporto o il loro forum per sapere se c'è un tale limite.
-
